src/Sinch/SinchClient.cs

```diff
@@ -131,7 +162,7 @@ public class SinchClient : ISinch
         internal SinchClient(string projectId, Uri authUri, Uri numbersBaseAddress, Uri smsBaseAddress)
         {
             var http = new HttpClient();
-            Auth = new Auth.Auth(authUri, http);
+            Auth = new Auth.OAuth(authUri, http);
```

Comment:
The call from this function contains comment `For E2E tests only. Here you can override base addresses.`
Ideally a dedicated function used for test only should be present at this level 
Official released product will contains dedicated to test only function but should not

Reply:
The function is not present in available public api as it's internal, and footprint in the final assembly is negligible. Not worth seating adjusting the build pipeline to remove the constructor.